### ISSUE TITLE
Background color of tabs in left side or right side configuration should match the theme

### ISSUE THREAD

**Issue by rmatte** [Eugeny/tabby#8252]
**Is your feature request related to a problem? Please describe.**
When configuring the tabs to be on the left or right side and using a dark theme the background of the tabs area is always pitch black which clashes with most themes and just doesn't look great.

**Describe the solution you'd like**
The background of the tab area in a right or left tab configuration should be based on the theme colors and blend in with it.

**Describe alternatives you've considered**
Not applicable.

**Additional context**
No additonal context to provide at this time.

**Comment by schultzm** [comment]
I have found when you click on settings > window > and change the dropdown, it seems to fit closer to the chosen color scheme
<img width="799" alt="Screenshot 2023-04-20 at 10 24 04" src="https://user-images.githubusercontent.com/5474971/233226648-d16cce66-5f6f-4e7b-8924-9e13de4bb32e.png">

**Comment by rmatte** [comment]
Yes, the legacy options do generally have a color tone that's closer to some of the themes but they are marked as legacy for a reason. The new "Follow the color scheme" should have an option to be able to configure the background color of the entire window in addition to the background color of the individual terminals and such. It would all fit together much better if it offered that type of flexibility.

**Comment by chrismyers81** [comment]
For the "Follow the color scheme" option, would it be possible to have it follow the color scheme for the individual profile, rather than for Tabby itself? The reason I ask is one of the most useful features of any SSH client (to me) is the ability to do a per-connection color scheme. That way my production connections can always have a red background, and non-production always have green.

**Comment by MajorTomDE** [comment]
+1
